src/main/scala/com/scalapenos/riak/internal/RiakHttpSupport.scala

```diff
@@ -38,6 +39,22 @@ private[riak] trait RiakUriSupport {
   }
 
   // ==========================================================================
+  // Conditional Request Parameters Support
+  // ==========================================================================
+
+  implicit class ConditionalHttpRequestParam(conditionalParam: ConditionalRequestParam) {
+    def asHttpHeader: HttpHeader = {
+      conditionalParam match {
+        case IfModifiedSince(date)   ⇒ `If-Modified-Since`(toSprayDateTime(date))
+        case IfUnmodifiedSince(date) ⇒ `If-Unmodified-Since`(toSprayDateTime(date))
+        case IfMatch(eTag)           ⇒ RawHeader("If-Match", eTag.value) // TODO this `If-Match`(EntityTag(eTag)) doesn't work as spray escapes double quotes in ETag value
+        case IfNotMatch(eTag)        ⇒ RawHeader("If-None-Match", eTag.value) // TODO this `If-None-Match`(EntityTag(eTag)) doesn't work as spray escapes double quotes in ETag value
```

Comment:
The TODO comments are slightly confusing. 
Are you saying that the "normal" Spray support for these two headers doesn't work because of an escaping problem and that you are therefor using raw headers? If so, fine because the functionality works. If not, than the PR isn't finished yet, right?